### PR TITLE
Update bridged_USDC_standard.md

### PR DESCRIPTION
**Pull request by blincf**
v2 of the doc. Updates specifically on the contract details and the token roles.

## Summary

Added more clarity on contract details and token roles in this v2 of the doc.

## Detail

As summary.

## Testing

NA

## Documentation

That's this PR's main change.

---

### CODE REVIEW

File: doc/bridged_USDC_standard.md [hunk 1]
```diff
@@ -16,22 +16,46 @@ This document provides a high-level overview of the process. Note that this also
    - Circle and the third-party team will jointly coordinate to burn the supply of native USDC locked in the bridge contract on the origin chain and upgrade the bridged USDC token contract on the destination chain to native USDC.
 4. The native USDC token contract seamlessly retains the existing supply, holders, and app integrations of the original bridged USDC token contract.
 
-### Bridge Contracts
+## Bridge Contracts
 
-The third-party team’s bridge contracts must be upgradeable to add required, specific functionality at a later point in time to support the upgrade steps above, once Circle and the third-party team jointly decide to perform the upgrade.
+The third-party team’s bridge contracts play an integral role and require small additions to support the upgrade process.
```
Comment (yvonnezhangc) [inline doc/bridged_USDC_standard.md:21]: Q: are we intentionally removing source bridge upgradeability as a requirement?
If not, I feel like this is still an important detail to call out. We should considering specifying the defined interface for `burnAllLockedUSDC` as well.

File: doc/bridged_USDC_standard.md [hunk 2]
```diff
@@ -59,13 +83,36 @@ Note that the text shown in parentheses above would be the third-party team’s
...
+
+If they are assigned to a smart contract, the contract must expose a function that Circle can call through a smart contract interaction to perform the role transfer at upgrade time. Automating this interaction can reduce manual errors and can enable atomic before and after checks that facilitate a smooth transfer. 
+
+This function must have the following signature: 
+
+```
+function transferUSDCRoles(address owner) external;
+```
+
+The function implementation details are left up to the partner, but it must: 1) only be callable by a Circle-owned address and 2) transfer the Implementation Owner role and the ProxyAdmin role to the function caller (if both are assigned to the bridge).
```
Comment (yvonnezhangc) [inline doc/bridged_USDC_standard.md:113]: Based on how the interface is defined, I believe the ProxyAdmin role will be assigned to the function caller, where as the new owner address should be specified in the function parameter.

``` function transferUSDCRoles(address owner) ```

**Review by yvonnezhangc — CHANGES_REQUESTED** [review]
Thanks so much for updating the doc! Two minor comments.

File: doc/bridged_USDC_standard.md (hunk 1 above)
Comment (blincf) [reply doc/bridged_USDC_standard.md:21]: The requirement is specified below on `burn`, but my understanding is that this doc intends to clarify things before they start deploying, and we recommend them to add the burn functionality after we mutually agree. So they aren't missing out this particular function name particularly. The following two lines hinted some on this regard.

File: doc/bridged_USDC_standard.md (hunk 2 above)
Comment (blincf) [reply doc/bridged_USDC_standard.md:113]: Yes in some ways we separate these into two function calls (they transfer the proxyAdmin to us first, and they call this function to transfer the owner role to us which we move the remaining roles).
The goal for this doc isn't to specify the upgrade process which often times requires more digging and isn't something we want every team to be ready to do before we agree to convert. What we can do is to make sure that they don't hit the problems when deploying bridged token that makes it impossible for future upgrade. This doc shouldn't describe the upgrade process in that level of detail.
Do you agree @mauriziocircle

**Review by yvonnezhangc — APPROVED** [review]

**Pull request merged**